core/jvm-native/src/main/scala/zio/internal/ZScheduler.scala

```diff
@@ -317,7 +317,10 @@ private final class ZScheduler(autoBlocking: Boolean) extends Executor {
                       if (nRunnables > 0) {
                         val iter = runnables.iterator
                         runnable = iter.next()
-                        if (nRunnables > 1) localQueue.offerAll(iter, nRunnables - 1)
+                        if (nRunnables > 1) {
+                          val iterable = iter.toIterable
```

Comment:
Where is `iterable` used? If nowhere, why not `val _ = ...`?

Reply:
Yes, can make it though 😅